Build.ps1

```diff
@@ -131,37 +125,37 @@ Find-SitecoreVersions -Path $imagesPath -InstallSourcePath $InstallSourcePath -F
         $sourceItem = Get-Item -Path $sourcePath
         $targetPath = Join-Path $version.Path $sourceItem.Name
 
-        if (!(Test-Path -Path $targetPath))
-        {
+        if (!(Test-Path -Path $targetPath)) {
             Copy-Item $sourceItem -Destination $targetPath -Verbose:$VerbosePreference
         }
     }
     
     # Build image
-    if ($tag -like "*SQL*")
-    {
+    if ($tag -like "*SQL*") {
         # Building SQL based images requires more memory than the default 2GB
         docker image build --isolation "hyperv" --memory 4GB --tag $tag $version.Path
     }
-    else
-    {
+    else {
         docker image build --isolation "hyperv" --tag $tag $version.Path
     }
 
     $LASTEXITCODE -ne 0 | Where-Object { $_ } | ForEach-Object { throw ("Build of '{0}' failed" -f $tag) }
 
     # Determine if we need to push
     $currentDigest = (docker image inspect $tag) | ConvertFrom-Json | ForEach-Object { $_.Id }
+    
+    if ($RemoveInstallationSourceFiles) {
+        Write-Host "Done with Installation Source - Removing  $targetPath" -ForegroundColor Green
```

Comment:
Please change to `Write-Host ("Done with Installation Source, removing '{0}'." -f  $targetPath) -ForegroundColor Green` to keep output messages consistent, ie. single quotes around important values and ending a message with either . or ...